misk/src/main/kotlin/misk/tracing/backends/jaeger/JaegerBackendModule.kt

```diff
@@ -0,0 +1,36 @@
+package misk.tracing.backends.jaeger
+
+import com.google.common.util.concurrent.Service
+import com.google.inject.Provides
+import com.google.inject.Singleton
+import com.uber.jaeger.Configuration
+import io.opentracing.Tracer
+import misk.config.AppName
+import misk.inject.KAbstractModule
+import misk.inject.addMultibinderBinding
+import misk.inject.to
+import misk.tracing.TracingService
+
+class JaegerBackendModule(val config: JaegerBackendConfig?) : KAbstractModule() {
+  override fun configure() {
+    binder().addMultibinderBinding<Service>().to<TracingService>()
+  }
+
+  @Provides
+  @Singleton
+  fun jaegerTracer(@AppName appName: String): Tracer {
+    val reporter =
+        if (config?.reporter == null) Configuration.ReporterConfiguration()
```

Comment:
could you default `reporter` to a new instance?

Reply:
`config.reporter` is a different type from the object being instantiated here. Changing the type on the config would break loading from the yaml